modules/generator/localserieslimiter/local_series_limiter.go

```diff
@@ -76,6 +77,10 @@ func New(maxSeriesFunc func(tenant string) uint32, tenant string, limitLogger *t
 	}
 }
 
+func (l *LocalSeriesLimiter) SanitizeLabels(lbls labels.Labels) labels.Labels {
+	return lbls
```

Comment:
Should the above logic be pulled out of the local entity limiter and added here as well?